Revise Contributing topic to provide only MAPDL-specific info (#829)
pyansys/pymapdl: doc/source/contributing.rst

    pip install -e .

Posting Issues
--------------
Use the `PyMAPDL Issues <https://github.com/pyansys/pymapdl/issues>`_
page to submit questions, report bugs, and request new features. When possible, we
recommend that you use these issue templates:

* Bug report template
* Feature request template

If your issue does not fit into one of these categories, create your own issue.

> To reach the project support team, email `pyansys.support@ansys.com <pyansys.suport@ansys.com>`_.

Author: ```suggestion
To reach the project support team, email `pyansys.support@ansys.com <pyansys.support@ansys.com>`_.
```

> To reach the project support team, email `alexander.kaszynski@ansys.com <alexander.kaszynski@ansys.com>`_.

Reviewer: Use pyansys.support@ansys.com instead please.

> To reach the project support team, email `alexander.kaszynski@ansys.com <alexander.kaszynski@ansys.com>`_.

Author: ```suggestion
To reach the project support team, email `pyansys.support@ansys.com <pyansys.suport@ansys.com>`_.
```

Viewing PyMAPDL Documentation
-----------------------------
Documentation for the latest stable release of PyAEDT is hosted at

> `PyMAPDL Documentation <https://mapdldocs.pyansys.com>`_.  
> 

Reviewer: ```suggestion
`PyMAPDL Documentation <https://mapdldocs.pyansys.com>`_, and latest development documentation at `PyMAPDL Development Documentation <https://dev.mapdldocs.pyansys.com>`_.  

```

      LEVEL - INSTANCE NAME - MODULE - FUNCTION - MESSAGE
      INFO - GRPC_127.0.0.1:50052 -  test - <module> - This is an useful message

The ``instance_name`` field depends on the name of the MAPDL instance, which might not be set
yet when the log record is created (for example, during the initialization of the library).
If an MAPDL instance is not yet created, this field might be empty.

Because both types of loggers are based in the Python module ``logging``, you can use any of
the tools provided in this module to extend or modify these loggers.

> For more information, see the module ``logging`` under ``ansys.mapdl.core``. 

Reviewer: @germa89, provide a link to our logging documentation API.

Reviewer: Done.
